### PR TITLE
Adding approved Code of Conduct Committee changes

### PR DESCRIPTION
**Pull request by amye**
* Charter
* Jurisdiction policy
* Resolution procedures

### CODE REVIEW

**Comment by amye** [comment]
@jeefy  - ready for your review

File: code-of-conduct/coc-committee-charter.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# CNCF Code of Conduct Committee Charter
+
+## Purpose
+
+The CNCF Code of Conduct Committee (“CoC Committee”) responds to, investigates, and resolves [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md) incidents.
+
+The Jurisdiction Policy describes the CoC Committee’s relationship to the Linux Foundation (LF) staff, who enforces the [Linux Foundation Events Code of Conduct](https://events.linuxfoundation.org/about/code-of-conduct/), and project-level Code of Conduct responders (e.g., the Kubernetes Code of Conduct Committee).
```
Comment (dims) [inline code-of-conduct/coc-committee-charter.md:7]: enforces -> enforce (i think!)

File: code-of-conduct/coc-committee-charter.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+## Composition
+
+The CoC Committee consists of 3 elected community members and 2 CNCF appointed staff members, plus alternates.
```
Comment (dims) [inline code-of-conduct/coc-committee-charter.md:11]: Do we have a count of alternates?

**Comment by amye** [comment]
Note: this was approved by the GB on July 13th, these were published in the Code of Conduct working group repo here: https://github.com/cncf/wg-coc

**Pull request merged**